CIP-0068/README.md

```diff
@@ -191,7 +191,7 @@ metadata =
 ; A valid Uniform Resource Identifier (URI) as a UTF-8 encoded bytestring.
 ; The URI scheme must be one of `https` (HTTP), `ipfs` (IPFS), `ar` (Arweave) or `data` (on-chain).
 ; Data URLs (on-chain data) must comply to RFC2397.
-uri = bounded_bytes ; UTF-8
+uri = bounded_bytes / [ * bounded_bytes ] ; UTF-8
```

Comment:
Why not use `bytes(*)` instead of `array(*)` in the first place?

Reply:
To add to @mmahut question:

Lucid does encode long strings as bytes(*) (using `Data.fromJson(...)`).
```
// excerpt from cbor.me of an image uri encoded as bytes(*)
  5F                             # bytes(*)
            58 40                       # bytes(64)
               697066733A2F2F516D5543584D546376754A7077484633674142527236396365515232754547324673696B3943795768384D556F51766572796C6F6E67757269 # "ipfs://QmUCXMTcvuJpwHF3gABRr69ceQR2uEG2Fsik9CyWh8MUoQverylonguri"
            58 19                       # bytes(25)
               646566696E6974656C796D6F72657468616E36346279746573 # "definitelymorethan64bytes"
            FF                          # primitive(*)
```

Does the change proposed in this PR mean that array(*) should be use instead?
```
// excerpt from cbor.me of an image uri encoded as array(*)
 9F                             # array(*)
            58 40                       # bytes(64)
               68747470733A2F2F6173736574317870743436647A336E39377979336C3937633035666A6C73727976357267677274306E6361372E6170652E6E667463646E2E # "https://asset1xpt46dz3n97yy3l97c05fjlsryv5rggrt0nca7.ape.nftcdn."
            58 40                       # bytes(64)
               696F2F696D6167653F73697A653D32353626746B3D506E307935746651766165654C6A4277764C646A506158684A322D445F735352664E4A6856386854336C6F # "io/image?size=256&tk=Pn0y5tfQvaeeLjBwvLdjPaXhJ2-D_sSRfNJhV8hT3lo"
            FF                          # primitive(*)
```

Reply:
Both bytes(*) and array(*) should be fine. In both instances it splits the values up into 64 byte chunks.

Reply:
A few people including myself assumed this was already part of the standard since it works for CIP-25 as well and many dapps already support the array format

Reply:
I am not good at reading CDDL, does `[ * bounded_bytes ]` allows for both indefinite byte string (5F) and indefinite array (9F) tags?

Since [ref implementation](https://github.com/spacebudz/ref-impl-cip68/) is using lucid, which implements this using indefinite byte string (5F), shouldn't that be the preffered way?

Reply:
CDDL does not really have a concept of a "chunked, indefinite length string".

Essentially the expected behavior for a decoder would be to convert from an array into a concatenated string whether an indefinite byte string or an array of max-length 64 byte strings...

As Nick mentioned, this is how it has "always worked" since the early days of CIP-25 and my presumption is that most decoders are already accounting for this possible scenario. Adding it as an option to the spec I don't think warrants a version bump as I'm fairly confident we could find multiple instances of this already being used and supported even though not officially documented.

Reply:
> CDDL does not really have a concept of a "chunked, indefinite length string".

Ah, interesting and rather disappointing. Thanks for the insight.

> Essentially the expected behavior for a decoder would be to convert from an array into a concatenated string whether an indefinite byte string or an array of max-length 64 byte strings...
>
> As Nick mentioned, this is how it has "always worked" since the early days of CIP-25 and my presumption is that most decoders are already accounting for this possible scenario. Adding it as an option to the spec I don't think warrants a version bump as I'm fairly confident we could find multiple instances of this already being used and supported even though not officially documented.

What about other fields such as `name` and `description`. I would assume the same presumption apply?

Reply:
>  I'm fairly confident we could find multiple instances of this already being used and supported even though not officially documented (...) 

The fact that various instance of this has been implemented doesn't mean we can break the standard. 

The standard says the URI is of `bounded_bytes` and changing that breaks the standard. 

For example, why Nick opened this PR in the first place is that Blockfrost honors this CIP (while other explorers do not). We are okay to implement this as CIP68v3.